cmd/nfd-master/main.go

```diff
@@ -122,6 +122,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 			"in the same format as in the config file (i.e. json or yaml). These options")
 	flagset.BoolVar(&args.EnableLeaderElection, "enable-leader-election", false,
 		"Enables a leader election. Enable this when running more than one replica on nfd master.")
+	flagset.Int64Var(&args.ListSize, "node-feature-informer-list-size", 0,
```

Comment:
I suggest a shorter way like this.
```suggestion
	flagset.Int64Var(&args.ListSize, "informer-list-size", 0,
```

Reply:
Sure thing

Reply:
Changed the flag and updated docs, let me know if you'd prefer the doc update somewhere else